libs/verifiable-credentials/src/main/java/uk/gov/di/ipv/core/library/verifiablecredential/helpers/VcHelper.java

```diff
@@ -225,6 +227,15 @@ private static Integer getAge(String dobValue) {
         }
     }
 
+    private static Object getPassportOrResPermitField(JSONObject credentialSubject) {
+        // If Passport not exist then try for ResidencePermit (BRP/BRC/FWP)
```

Comment:
Have you confirmed that all 3 of these look the same (i.e. use `residencePermit`)? It seems likely, but I've not checked myself.

Reply:
Which team can confirm that...who can share the sample VC for PP/BRP/BRC/FWP/DL

Reply:
I'd hope that it's in confluence somewhere, but otherwise you could try a BA, Simon, or asking the mobile team directly

Reply:
Found one link - https://govukverify.atlassian.net/wiki/spaces/~63170c3262fe1e6eac6d6c75/pages/3446636717/VC+Structures

ResidentPermit - in progress

Reply:
I suppose residence permit has 'documentType' to identify one type of residence permit from another.
e.g
"residencePermit": [
        {
          "icaoIssuerCode": "GBR",
          "documentNumber": "ZR8016200",
          "expiryDate": "2024-02-25",
          "documentType": "BRP"
        }
      ]

Reply:
Ok, well we can always merge and examine any remaining nulls further